include/nighthawk/common/rate_limiter.h

```diff
@@ -49,6 +49,8 @@ class DiscreteNumericDistributionSampler {
 public:
   virtual ~DiscreteNumericDistributionSampler() = default;
   virtual uint64_t getValue() PURE;
+  virtual uint64_t min() const PURE;
```

Comment:
Can we document the new methods?

(optional / unrelated) Since we are here, can we document the entire interface?